learn/get-started/write-a-graphql-api-with-ballerina.md

```diff
@@ -167,19 +194,21 @@ service /covid19 on new graphql:Listener(9000) {
 }
 ```
 
-In the above code snippet, the path of this service is defined as `/covid19`. If you want to host the service on the
-root, you can remove the path as the following code snippet:
+In this code:
+- The path of this service is defined as `/covid19`. If you want to host the service on the
+root, you can remove the path as the follows:
 
-```ballerina
-import ballerina/graphql;
+    ```ballerina
+    import ballerina/graphql;
 
-service on new graphql:Listener(9000) {
+    service on new graphql:Listener(9000) {
 
-}
-```
+    }
+    ```
+
+#### Define the port
```

Comment:
Incorrect title. This code block and section is not about defining port, but about how to use exiting listneer.See more comments below for this section,

Reply:
Removed.